docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+      quadrupolar={"Cq":1.72e6, "eta":0.5}  # Cq in Hz
+      )
+
+    # No Couplings, so create a separate SpinSystem for each site.
+    sites = [site1, site2, site3]
+    spin_systems = [SpinSystem(sites=[s]) for s in sites]
+
+    sim = Simulator(spin_systems=spin_systems, methods=[my_mqmas])
+    sim.run()
+
+    # Apply Gaussian line broadening along both dimensions
+    processor = sp.SignalProcessor(
+        operations=[
+            sp.IFFT(dim_index=(0, 1)),
+            sp.apodization.Gaussian(FWHM="0.08 kHz", dim_index=0),
+            sp.apodization.Gaussian(FWHM="0.22 kHz", dim_index=1),
+            sp.FFT(dim_index=(0, 1)),
+        ]
+    )
+    dataset = processor.apply_operations(dataset=sim.methods[0].simulation)
+
+
+In the code below, we use the PyPlot method `imshow()
+<https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.imshow.html>`__ to
+return an image of the dataset on a 2D regular raster. We also use `gist_ncar_r
+<https://matplotlib.org/stable/gallery/color/colormap_reference.html>`__ as the
+colormap to map the dataset amplitude to colors. The "_r" indicates a reversed
+version of the "gist_ncar" map. The `colorbar()
+<https://matplotlib.org/stable/api/colorbar_api.html?highlight=colorbar#module-matplotlib.colorbar>`__
+function provides the visualization of the dataset mapping to color to the right
+of the plot.
+
+
+.. skip: next
+
+.. plot::
+    :context: close-figs
+
+    plt.figure(figsize=(4, 3))
+    ax = plt.subplot(projection="csdm")
+    cb = ax.imshow(dataset.real / dataset.real.max(), aspect="auto", cmap="gist_ncar_r")
+    plt.colorbar(cb)
+    ax.invert_xaxis()
+    ax.invert_yaxis()
+    plt.tight_layout()
+    plt.show()
+
+.. warning::
+    This custom method, as well as the built-in Multi-Quantum VAS methods, assumes uniform
+    excitation and mixing of the multiple-quantum transition. In an experimental MQ-MAS
+    measurement, excitation and mixing efficiencies depend on the ratio of the quadrupolar
+    coupling constant to the rf field strength. Therefore, the relative integrated intensities
+    of this simulation may not agree with the experiment.
+
+
+Inspecting Transition and Symmetry Pathways
+'''''''''''''''''''''''''''''''''''''''''''
+
+You can view the symmetry pathways that will be selected by your custom method
+in a given spin system using the function
+:py:meth:`~mrsimulator.Method.get_symmetry_pathways` as shown below.
+
+.. plot::
+    :context: close-figs
+
+    from pprint import pprint
+    pprint(my_mqmas.get_symmetry_pathways("P"))
+    pprint(my_mqmas.get_symmetry_pathways("D"))
+
+.. rst-class:: sphx-glr-script-out
+
+ Out:
+
+ .. code-block:: none
+
+    [SymmetryPathway(
+        ch1(87Rb): [-3] ⟶ [-1]
+        total: -3.0 ⟶ -1.0
+    )]
+    [SymmetryPathway(
+        ch1(87Rb): [0] ⟶ [0]
+        total: 0.0 ⟶ 0.0
+    )]
+
+
+**Method** also has a related function :py:meth:`~mrsimulator.Method.plot` for generating a symmetry
```

Comment:
It might be useful to include a note that this function is under development since there are numerous planned changes for the next version. @pjgrandinetti @deepanshs thoughts?

Reply:
We can remove it.  I didn't realize until later that there was a problem with MixingEvents.  But I assume there is no problem with get_transition_pathways()?

Reply:
Yes, get_transition_pathways() works as expected